src/test/groovy/com/github/vkorobkov/jfixtures/AfterInsertsTest.groovy

```diff
@@ -0,0 +1,40 @@
+package com.github.vkorobkov.jfixtures
+
+import com.github.vkorobkov.jfixtures.testutil.YamlVirtualFolder
+import spock.lang.Specification
+
+import java.nio.file.Path
+
+class AfterInsertsTest extends Specification implements YamlVirtualFolder {
+    Path tmpDefaultFolderPath
+    Path tmpCustomFolderPath
+
+    def DEFAULT_EXPECTED_SQL = """DELETE FROM "users";
+            |INSERT INTO "users" ("id", "name", "age") VALUES (10000, 'Vladimir', 29);
+            |""".stripMargin()
+    def CUSTOM_EXPECTED_SQL = """DELETE FROM "users";
+            |INSERT INTO "users" ("id", "name", "age") VALUES (10000, 'Vladimir', 29);
+            |// Completed table users
+            |COMMIT TRANSACTION;
+            |""".stripMargin()
+
+    void setup() {
```

Comment:
Maybe it makes sense to remove shared `setup` method and shared `tmpDefaultFolderPath` and `tmpCustomFolderPath` fields and to make this job in test methods themself:
```java
setup:
def folder = unpackYamlToTempFolder("no_after_inserts.yml")

expect:
JFixtures.postgres(tmpDefaultFolderPath as String).asString() == DEFAULT_EXPECTED_SQL

cleanup:
folder.toFile().deleteDir()
```